ding/worker/replay_buffer/naive_buffer.py

```diff
@@ -134,6 +156,7 @@ def _append(self, ori_data: Any, cur_collector_envstep: int = -1) -> None:
             self._push_count += 1
             if self._data[self._tail] is None:
                 self._valid_count += 1
+                self._periodic_thruput_monitor.valid_count = self._valid_count
```

Comment:
You need update `self._periodic_thruput_monitor.valid_count` wherever `self._valid_count` is updated.